cli/src/api/ZoweCli.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * This program and the accompanying materials are made available and may be used, at your option, under either:
+ * * Eclipse Public License v2.0, available at https://www.eclipse.org/legal/epl-v20.html, OR
+ * * Apache License, version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+import { ZssRestClient } from "./ZssRestClient";
+import { AbstractSession, Logger, ImperativeExpect, Headers, RestClientError, ImperativeError } from "@zowe/imperative";
+export class ZoweCli {
+    static LoginPath = "/login";
```

Comment:
i need some explanation here: this is login code, but so is zowe/login/login.handler.ts
what is the difference?